lib/restforce/db/accumulator.rb

```diff
@@ -31,6 +31,19 @@ def attributes
         end
       end
 
+      # Public: Get a Hash representing the current values for the items in the
+      # passed Hash, as a subset of this Accumulator's attributes Hash.
+      #
+      # comparison - A Hash mapping of attributes to values.
+      #
+      # Returns a Hash.
+      def current(comparison)
+        attributes.each_with_object({}) do |(attribute, value), diff|
```

Comment:
Good catch. Re: the obsolete method, I sort of felt like it might be useful to keep around, but a combination of YAGNI and "it's already in the git history" have talked me out of that perspective.